client/src/app/EmptyTab.js

```diff
@@ -37,20 +35,16 @@ export default class EmptyTab extends PureComponent {
 
     return (
       <Tab className={ css.EmptyTab }>
-        <p className="create-buttons">
-          <span>Create a </span>
-          <button className="create-bpmn btn btn-secondary" onClick={ () => onAction('create-bpmn-diagram') }>BPMN diagram</button>
+        <div className="create-buttons">
+          <p>Create a new file:</p>
+          <button className="btn btn-secondary" onClick={ () => onAction('create-bpmn-diagram') }>BPMN diagram (Camunda Engine)</button>
+          <button className="btn btn-secondary" onClick={ () => onAction('create-cloud-bpmn-diagram') }>BPMN diagram (Zeebe Engine)</button>
           {
             !Flags.get(DISABLE_DMN) && (
-              <Fragment>
-                <span> or </span>
-                <button className="btn btn-secondary" onClick={ () => onAction('create-dmn-diagram') }>DMN diagram</button>
-              </Fragment>
+              <button className="btn btn-secondary" onClick={ () => onAction('create-dmn-diagram') }>DMN diagram (Camunda DMN Engine)</button>
             )
           }
-        </p>
-
-        <Slot name="empty-tab-buttons" />
```

Comment:
Fingers crossed there are no plugins using this slot.

But since we're not using it ourselves and didn't communicate this slot in the past, it's fine.

Reply:
This extension point was so secret that I didn't even know about it 🙈